tests/Nox.Types.Tests/Types/Guid/GuidTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿// ReSharper disable once CheckNamespace
+using FluentAssertions;
+
+namespace Nox.Types.Tests.Types;
+
+public class GuidTests
+{
+    [Fact]
+    public void FromString_Construct_ReturnNewGuid()
+    {
+        var textGuid = System.Guid.NewGuid().ToString();
+
+        var guid = Guid.From(textGuid);
+
+        guid.Should().NotBeNull();
+        guid.Value.Should().NotBeEmpty();
+        guid.ToString().Should().Be(textGuid);
+    }
+
+    [Fact]
+    public void Guid_ToString_ReturnSameGuid()
```

Comment:
This test is doing the same as the first one, can you merge them?

Reply:
Tests merged.